lib/ex_unit/test/ex_unit/filters_test.exs

```diff
@@ -5,6 +5,99 @@ defmodule ExUnit.FiltersTest do
 
   doctest ExUnit.Filters
 
+  describe "filter precedence" do
+    setup do
+      [
+        one: %{
+          one: true,
+          numeric: true,
+          test: :"test one",
+          test_type: :test
+        },
+        two: %{
+          skip: true,
+          numeric: true,
+          test: :"test two",
+          test_type: :test,
+          two: true
+        }
+      ]
+    end
+
+    test "no filters", tags do
+      assert ExUnit.Filters.eval([], [], tags.one, []) == :ok
+      assert ExUnit.Filters.eval([], [], tags.two, []) == {:skipped, "due to skip tag"}
+    end
+
+    test "exclude tests", tags do
+      assert ExUnit.Filters.eval([], [:two], tags.one, []) == :ok
+      assert ExUnit.Filters.eval([], [:test], tags.one, []) == {:excluded, "due to test filter"}
+      assert ExUnit.Filters.eval([], [:test], tags.two, []) == {:excluded, "due to test filter"}
+
+      assert ExUnit.Filters.eval([], [:numeric], tags.one, []) ==
+               {:excluded, "due to numeric filter"}
+
+      assert ExUnit.Filters.eval([], [:unknown], tags.one, []) == :ok
+      assert ExUnit.Filters.eval([], [:unknown], tags.two, []) == {:skipped, "due to skip tag"}
+    end
+
+    test "explicitly exclude a test with the `:skip` tag", tags do
```

Comment:
Minor nitpick: I would remove the backticks in the test description. 

There are only 4 places in all the codebase with backticks, and I just pushed a [PR](https://github.com/elixir-lang/elixir/pull/8343) to remove them 🙂

Reply:
It was me who made sure they were not there :-D
check also for comments, they should not be included either

Reply:
fixed. thank you for pointing that out